src/test/Axios.ts

```diff
@@ -0,0 +1,47 @@
+// npm run test -- --grep "Axios"
+
+import { createWalletFull, MoneroNetworkType, MoneroWalletListener } from "../..";
+import { connectToWalletRpc } from "../..";
+
+describe("Axios", function() {
```

Comment:
Thanks, but let's actually remove this test, since it's redundant with existing tests for monero-wallet-rpc in [TestMoneroWalletRpc.ts](https://github.com/woodser/monero-ts/blob/master/src/test/TestMoneroWalletRpc.ts).

Reply:
agree